Rawl_B502/genelach_dáil_cormaic_&_úa_labrada.ttl

```diff
@@ -0,0 +1,102 @@
+@prefix irishRel: <http://example.com/earlyIrishRelationship.ttl#> .
+@prefix irishTitle: <http://example.com/earlyIrishTitles.ttl#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+
+<>
+    a dctype:Dataset;
+    dcterms:title "Genelach Dáil Cormaic & Úa Labrada"@sga;
+    dcterms:isFormatOf <http://www.ucc.ie/celt/published/G105003/text003.html>;
+    dcterms:format "text/turtle".
+
+<#Corbmac>
+    a foaf:Person;
+    irishRel:nomName "Corbmac";
+    owl:sameAs <http://example.com/Rawl_B502/mínugud_senchusa_laigin_and_so_sís.ttl#Corbmac>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Cormac>;
+    rel:childOf <#ConCorb>;
+    irishRel:numChild 1.
+
+<#ConCorb>
+    a foaf:Person;
+    irishRel:nomName "Cú Corb";
+    irishRel:genName "Con Corb";
+    owl:sameAs <http://example.com/Rawl_B502/de_genelogia_dáil_coirpri_arad_retro.ttl#ConCorb>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#ConCorbb-d2451e3d>.
+
+<#Imchad>
+    a foaf:Person;
+    irishRel:nomName "Imchad";
+    rel:childOf <#Corbmac>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Imchad>.
+
+<#Labraid>
+    a foaf:Person;
+    irishRel:nomName "Labraid";
+    rel:childOf <#Imchad>;
+    rdfs:comment "ó tát Úi Labrada";
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Labraid>;
+    irishRel:numChild 7.
+
+<#Luguid>
+    a foaf:Person;
+    irishRel:nomName "Luguid";
+    rel:childOf <#Labraid>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Lugaid>.
+
+<#Cathbud>
+    a foaf:Person;
+    irishRel:nomName "Cathdub";
+    rel:childOf <#Labraid>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Cathub>.
+
+<#Cóeldub>
+    a foaf:Person;
+    irishRel:nomName "Cóeldub";
+    rel:childOf <#Labraid>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Caeldub>.
+
+<#Cainnech>
+    a foaf:Person;
+    irishRel:nomName "Cainnech";
+    rel:childOf <#Labraid>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Cainnech>.
+
+<#Cáirthenn>
+    a foaf:Person;
+    irishRel:nomName "Cáirthenn";
+    rel:childOf <#Labraid>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#Cáirthend>.
+
+<#NioCuillenn>
+    a foaf:Person;
+    irishRel:nomName "Nio Cuillenn";
+    rel:childOf <#Labraid>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#NioCulend>.
+
+<#DaigBec>
+    a foaf:Person;
+    irishRel:nomName "Daig Bec";
+    rel:childOf <#Labraid>;
+    owl:sameAs <http://example.com/LL/dal_corpri_arad.ttl#DiagBec>.
+
+<#Lugdach>
+    a foaf:Person;
+    irishRel:genName "Lugdach";
+    irishRel:nomName "Lugaid";
+    owl:sameAs <#Luguid>;
+    rel:ancestorOf <#Ochrai>.
+
+<#Ochrai>
+    a foaf:Person;
+    irishRel:genName "Ochrai";
+    rdfs:comment "Is sí clann Lugdach Úi Ochrai";
+    owl:sameAs <http://example.com/Rawl_B502/de_genealogia_úa_n-ochrae.ttlFergusaÓenochrae>;
+    rdfs:seeAlso <http://example.com/Rawl_B502/de_genealogia_úa_n-ochrae.ttl>.
```

Comment:
Pointing forwards. Basically, I was trying to reflext the fact that the text straddles out file boundary. This is what it looks like:
> ¶70]... Nio Cuillenn, Daig Bec
> ¶71] Is sí clann Lugdach Úi Ochrai.
> DE GENELOGIA ÚA N-OCHRAE.
> ¶72] Dub Dá Chrích mc h-Írgusa ... m. Lugdach ...
Basically, the last sentence of <Rawl_B502/genelach_dáil_cormaic_&_úa_labrada.ttl> introduces <http://example.com/Rawl_B502/de_genealogia_úa_n-ochrae.ttl>. But I have also made the people mentioned in this last sentence owl:sameAs the respective URIs in the next file. Perhaps this is sufficient linkage?